exercises/leap/canonical-data.json

```
@@ -1,49 +1,49 @@
 {
   "exercise": "leap",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

I think these changes would only be considered a minor change.
```suggestion
  "version": "1.5.1",
```

---

I considered that - expect your spec for version changes (the one mentioned in the linter, which failed for me) indicates that the final .x is for patch changes (and this isn't a patch change)? Happy to go with this, but we should also adjust those docs too.

---

I am not sure I understand what you are saying. I can see that Travis CI failed after the 2nd commit to this PR because "version" was not changed.  Branch `leap-year-code-gen-improvement` maybe should have been rebased onto master after #1465 was merged, it looks like master was merged into `leap-year-code-gen-improvement` instead.  This is when Travis CI failed.  `bin/check_versions` only confirms if the version has been changed.  I assume this is what you are speaking about; the third commit reflects the version change.  So far this PR has only changed the descriptions which is defined as a [patch change](https://github.com/exercism/problem-specifications#patch-version-changes).  If `leapYear` becomes `isLeapYear` that would constitute a [major version](https://github.com/exercism/problem-specifications#major-version-changes) change.

---

Sorry it was a bit late and I was too brief - when I talked about "patch" I wasn't referring to my branch mis-step - I was referring to the very nice lint checking error message I got: `You can see the versioning policy at https://github.com/exercism/problem-specifications#test-data-versioning`.  In there is talks about the patch version (that last digit). But reviewing it again, I see I missed the last part of the sentence where its not just for bug fixes but 
"whenever the meaning of the tests does not change.".  I didn't read it properly, and 1.5.1 is better.

Thanks for your patience on all this. In retrospect however, I think these newances aren't worth the effort for either of us - we have much bigger fish to fry.  I 'll finish this one but will hold back on future ones.